enterprise/frontend/src/embedding-sdk/README.md

```diff
@@ -499,6 +499,30 @@ Dashboards that support editing if a user has permissions for this, could be emb
   but without its cards - at this stage dashboard title, tabs and cards grid is rendered, but cards content is not yet
   loaded.
 
+### Creating a Question
+
+With the `NewQuestion` component, you can create a new question from scratch using the Metabase Notebook Editor. 
+
+#### Parameters
+
+No parameters are required for this component.
+
+```tsx
+import React from "react";
+import { MetabaseProvider, NewQuestion } from "@metabase/embedding-sdk-react";
+
+const config = {...}
+
+export default function App() {
+  return (
+    <MetabaseProvider config={config}>
+        <NewQuestion />
```

Comment:
Also since we have a collection picker in this PR when you select where you want to save the question. To be consistent, I think we should support passing `collectionId` the same way our collection browser does to specify the root collection the user could see so the APIs are consistent.

If that's something that needs more work then it's obviously could be done in a separate PR.

Reply:
yeah I'd like to do a follow-up PR once the main functionality for this one and Modify Question are done so we can look at both of them and figure out what props we'll want